pkg/kubevirt/machine.go

```diff
@@ -115,30 +143,39 @@ func (m *Machine) Address() string {
 }
 
 // IsBooted checks if the VM is booted.
-func (m *Machine) IsBooted(executor CommandExecutor) bool {
-	if m.Address() == "" {
-		return false
-	}
-
-	output, err := executor.ExecuteCommand("hostname")
-	if err != nil || output != m.machineContext.KubevirtMachine.Name {
+func (m *Machine) IsBooted() bool {
+	if !m.hasReadyCondition() {
```

Comment:
I suggest replacing IsBooted function with HasReadyCondition, and not calling one function in the other
Because this is not exactly the same and might confuse

Reply:
or maybe just calling it `IsReady()`?

Reply:
+1 for `isReady()`

Reply:
done!